src/components/CertificationMetadataForm/index.tsx

```diff
@@ -58,6 +58,7 @@ const CertificationMetadataForm = (props: Props) => {
         ...request,
         certificationLevel: !props.isReviewCertification ? parseInt(request.certificationLevel) : undefined,
         report: !props.isReviewCertification ? request.report.map(report => report.value) : undefined,
+        subject: props.isReviewCertification && profile?.dapp?.subject ? undefined : request.subject
```

Comment:
Can you elaborate a bit on the condition used here (preferably with a comment in the code)? 
Why If there is a subject in the `Dapp` profile we use `undefined` for the request and only when there is nothing we use what the user filled in?

Reply:
Here we tried to use the same CertificationMetadataForm component for 'Auditor Report Upload' and 'Review Certification Metadata'

Reply:
This was done with the intent to not include the property in the request. I've updated it to a different piece of code now